pkg/cmd/clientConfig.go

```diff
@@ -82,10 +81,13 @@ kubectl plugin mobile get clientconfig`,
 				}
 				ret = append(ret, svcConfig)
 			}
+
 			outputJSON := ServiceConfigs{
-				Services:  ret,
-				Namespace: ns,
-				ClientID:  clientID,
+				Version:     "1.0",
```

Comment:
Is is possible to have version as number. Like: `Version: 1`
Every version will be breaking here so 1.1 gives 0 value, but this will require additional parsing on clients.